### ISSUE TITLE
WindowsError: [Error 6] when using from GUI app

### ISSUE THREAD

**Issue by alexbruy** [workhorsy/py-cpuinfo#70]
I use py-cpu info in GUI app to retrieve information about processor. It works fine on Linux and Mac, but on Windows I get this error
```
WindowsError: [Error 6] The handle is invalid 
Traceback (most recent call last):
File "C:/Users/qgis/.qgis2/python/plugins\reportingtool\plugin.py", line 97, in run
report, filePath = createReport()
File "C:\Users\qgis\.qgis2\python\plugins\reportingtool\ext-libs\qgissysinfo\createreport.py", line 49, in createReport
report = qgissysinfo.info_as_text()
File "C:\Users\qgis\.qgis2\python\plugins\reportingtool\ext-libs\qgissysinfo_init_.py", line 31, in info_as_text
return _as_text(info())
File "C:\Users\qgis\.qgis2\python\plugins\reportingtool\ext-libs\qgissysinfo_init_.py", line 8, in info
_info = systeminfo.allSystemInfo()
File "C:\Users\qgis\.qgis2\python\plugins\reportingtool\ext-libs\qgissysinfo\systeminfo.py", line 62, in allSystemInfo
info = systemInfo()
File "C:\Users\qgis\.qgis2\python\plugins\reportingtool\ext-libs\qgissysinfo\systeminfo.py", line 80, in systemInfo
"Processor": cpuinfo.get_cpu_info()['brand'],
File "C:\Users\qgis\.qgis2\python\plugins\reportingtool\ext-libs\cpuinfo\cpuinfo.py", line 1787, in get_cpu_info
CopyNewFields(info, _get_cpu_info_from_cpuid())
File "C:\Users\qgis\.qgis2\python\plugins\reportingtool\ext-libs\cpuinfo\cpuinfo.py", line 1121, in _get_cpu_info_from_cpuid
returncode, output = run_and_get_stdout([sys.executable, "-c", "import cpuinfo; print(cpuinfo.actual_get_cpu_info_from_cpuid())"])
File "C:\Users\qgis\.qgis2\python\plugins\reportingtool\ext-libs\cpuinfo\cpuinfo.py", line 203, in run_and_get_stdout
p1 = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
File "C:\PROGRA~1\BOUNDL~1\Desktop\1.0\osgeo4w\apps\Python27\lib\subprocess.py", line 703, in _init_
errread, errwrite) = self._get_handles(stdin, stdout, stderr)
File "C:\PROGRA~1\BOUNDL~1\Desktop\1.0\osgeo4w\apps\Python27\lib\subprocess.py", line 839, in _get_handles
p2cread = self._make_inheritable(p2cread)
File "C:\PROGRA~1\BOUNDL~1\Desktop\1.0\osgeo4w\apps\Python27\lib\subprocess.py", line 878, in _make_inheritable
_subprocess.DUPLICATE_SAME_ACCESS)
WindowsError: [Error 6] The handle is invalid
```

**Comment by workhorsy** [comment]
What version of python are you using? Is this pythonw.exe?

Thanks for the bug report.

**Comment by alexbruy** [comment]
I use Python 2.7.5. It is python.exe.

**Comment by workhorsy** [comment]
I just tried reproducing by making a plugin with QGIS. It works fine for me. Yet I can find bug reports of people still having this problem. Such as https://bugs.python.org/issue3905.

My guess is that it is a Python bug caused by the GUI program starting without stdin. Apparently there is a workaround like this. Even tough that should not make any difference:
```python
# Changed this
p1 = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)

# To this
p1 = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE)
```

**Comment by workhorsy** [comment]
Can you try the current master and see if it works please?

**Comment by alexbruy** [comment]
Tested latest master and it works fine for me. Thanks!

**Comment by workhorsy** [comment]
Excellent. I was worried that it was caused by something else.

**Comment by workhorsy** [comment]
Fixed in commit: 47bcc464592f77425cd0daf13498b0a1108a2448
